src/common/utils.js

```diff
@@ -12,6 +12,10 @@ function replaceRollsCallback(match, replaceCB) {
     return result;
 }
 
+function getDieRollResult(faces) {
+    return Math.floor(Math.random() / (1 / faces)) + 1
```

Comment:
That's in no way better, you're changing `x * y`  into `x / (1 / y)` which is the exact same mathematical equation.
The trick with the division in the examples would only work for integer random values between 0 and 2**32, not for floating points between 0 and 1.0.
See the suggested link which makes it unbiased by making the range into an integer and discarding values that fall in that small interval between where the modulo wouldn't be uniform anymore. https://pthree.org/2018/06/13/why-the-multiply-and-floor-rng-method-is-biased/

Also, I'd prefer if this function was a method inside the `DNDBDice` class, instead of being a function in the utils file.

Reply:
Yeah, I figured I probably should have googled it rather than just make something up.

When I first wrote it, I noticed it was being used everywhere so I wanted to keep it in a common place. After updating it to use the function I then realised that most of those duplicates were in the dist folder, and weren't _really_ duplicates.

I'll have a look at that link.